src/main/java/rx/observables/StringObservable.java

```diff
@@ -355,24 +355,49 @@ public String call(Object obj) {
         });
     }
 
-    /**
+ /**
```

Comment:
Indentation missing here. Could you also squash the commits into just a single one?